pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1318,8 +1320,9 @@ func (ctrl *PersistentVolumeController) provisionClaimOperation(claimObj interfa
 		ctrl.eventRecorder.Event(claim, v1.EventTypeWarning, "ProvisioningFailed", strerr)
 		return
 	}
-
+	start := time.Now()
 	volume, err = provisioner.Provision()
+	metrics.VolumeProvisionLatency.Observe(metrics.SinceInMilliseconds(start))
```

Comment:
Same as before, it might be useful to know which provisioner/plugin is doing this provisioning  and that information should be available as a dimension.